scenarios/vault-consul-storage/hashibox.yaml

```diff
@@ -265,6 +265,12 @@ provision:
 
       touch /shikari-bootstrapped
 
+copyToHost:
```

Comment:
For this scenario you should be copying the Vault CA file and not Consul and Nomad.

Reply:
Since consul and vault are both using Consul CA, Only added one CACERT  on host machine, else will be a conflict

Reply:
Pls ignore previous comment. The vault cert has been added